### ISSUE TITLE
GPU wizards and clusters/queues....

### ISSUE THREAD

**Issue by pconesa** [I2PC/scipion#1820]
**Is your feature request related to a problem? Please describe.**
Trying to run gAutomatch wizard in a server where GPUs are only available after the queue.

**Describe the solution you'd like**
GPU wizard to work, using the queue? (hard, I know) 

**Describe alternatives you've considered**
Otherwise tell me that is not available if there is no GPU.

**Comment by azazellochg** [comment]
In my opinion, wizards are by design interactive, if you don't have a local GPU, submit the job to the queue then. I don't see the reason to implement this.

**Comment by delarosatrevin** [comment]
I agree with @azazellochg. One option is to submit an interactive job into the queue, log into the assigned node, launch the wizard interactively there...and let the job running (or re-submit into more nodes once the parameters are tuned)

**Comment by pconesa** [comment]
then, do not show the wizzard? Because now wizzard is shown, picking windows pop up but nothing happens. The user does not know is parameters are wrong or is not working.

**Comment by azazellochg** [comment]
How this can be implemented?.. check if CUDA=True in config file?

**Comment by pconesa** [comment]
There are Pyrrhon modules that can check if there are GPUs available

**Comment by azazellochg** [comment]
@delarosatrevin I assigned you since I dont know about such python modules...

**Comment by pconesa** [comment]
I think, scipion should provide that information to the plugin first, and then wizards, maybe, have a flag? needsGPU?
